scripts/cli.ts

```diff
@@ -0,0 +1,32 @@
+import { Option, program } from '@commander-js/extra-typings'
+import { sendAuditDataAction } from './manualAuditDataRequests/sendResults/sendAuditDataAction'
+import { AWS_REGION } from './utils/constants'
+
+process.env.AWS_REGION = AWS_REGION
+
+program
+  .name('ticf-integration-cli')
+  .description('cli tool for txma ticf integration')
+
+program
```

Comment:
Adding prompts to these scripts could be useful, or an example to copy and paste from the readme